lib/devicectl.js

```diff
@@ -246,4 +255,37 @@ export class Devicectl {
     });
     return JSON.parse(stdout).result.apps;
   }
+
+  /**
+   * Launch the given bundle id application with the given environment variable.
+   * This method is over devicectl command, this it may take additional seconds to launch the app.
+   * Please use via WDA or via appium-ios-device as primary method to launch app if possible.
+   *
+   * @param {LaunchAppOptions} opts launching app with devicectl command options.
+   * @returns {Promise<void>}
+   * @throws {Error} If the launching app command fails. For example, the given bundle id did not exist.
+   */
+  async launchApp(opts) {
+    const {
+      bundleId,
+      env,
+      terminateExisting = false
+    } = opts;
+
+    if (!_.isString(bundleId)) {
```

Comment:
I would say it still makes sense to extract bundleId from opts to reflect the fact it's required. Also, then we may drop this check.

Reply:
I follow the same strategy in other API, like pullFile, where from/to are required arguments beside options

Reply:
Sure. sounds good. Let me modify in wda side and push it, then bump the version with the change